tensorflow/training/docker/2.7/py3/cu112/Dockerfile.sagemaker.gpu

```diff
@@ -0,0 +1,42 @@
+ARG BASE_IMAGE=""
+FROM $BASE_IMAGE
+
+ARG SMDEBUG_VERSION=1.0.12
+ARG PYTHON=python3.8
+
+RUN $PYTHON -m pip install --no-cache-dir -U \
+    "sagemaker>=2,<3" \
+    sagemaker-experiments==0.* \
+    "sagemaker-tensorflow>=2.6,<2.7" \
+    "sagemaker-tensorflow-training>=20" \
+    "sparkmagic<1" \
+    "sagemaker-studio-sparkmagic-lib<1" \
+    "sagemaker-studio-analytics-extension==0.0.2" \
+    "smdebug==${SMDEBUG_VERSION}" \
+    smclarify 
+
+RUN $PYTHON -m pip install --no-cache-dir -U \
+    "bokeh>=2.3,<3" \
+    "imageio>=2.9,<3" \
+    "opencv-python>=4.3,<5" \
+    "plotly>=5.1,<6" \
+    "seaborn>=0.11,<1" \
+    "numba<0.54" \
+    "shap>=0.39,<1"
+
```

Comment:
Boost is missing:
https://github.com/aws/deep-learning-containers/blob/master/tensorflow/training/docker/2.6/py3/cu112/Dockerfile.gpu#L135

Reply:
You can check your TF compile flag using the following way and decide which flag to use
```
wget https://sourceforge.net/projects/boost/files/boost/1.73.0/boost_1_73_0.tar.gz/download -O boost_1_73_0.tar.gz
tar -xzf boost_1_73_0.tar.gz
cd boost_1_73_0
./bootstrap.sh
ABI_FLAG=$(python3 -c "import tensorflow as tf; ABI=[x for x in tf.sysconfig.get_compile_flags() if 'CXX11_ABI' in x]; print(' '.join(ABI), end='')")
if [[ "$ABI_FLAG" == "-D_GLIBCXX_USE_CXX11_ABI=0" ]]; then
  ./b2 define=_GLIBCXX_USE_CXX11_ABI=0 threading=multi --prefix=/usr -j 64 cxxflags=-fPIC cflags=-fPIC install || true
else
  ./b2 threading=multi --prefix=/usr -j 64 cxxflags=-fPIC cflags=-fPIC install || true
fi
cd ..
```